.github/mergify.yml

```diff
@@ -105,3 +105,39 @@ pull_request_rules:
       comment:
         message: |
           This PR changes implementation code, but doesn't include a changeset. Did you forget to add one?
+
+  - name: Add indexer tag and ecopod reviewers
+    conditions:
+      - 'files~=^indexer/'
+    actions:
+      label:
+        add:
+          - indexer
+      request_reviews:
+        users:
```

Comment:
@roninjin10 an alternative, that might be easier to maintain could be to create a team in GH for ecosystem and use `team` instead of `users` - something like

```
...
request_reviews:
   teams:
   - "@ethereum-optimism/ecopod"
```

Reply:
Didn’t know you could do that it’s a good idea!